lib/trento/event_handlers_supervisor.ex

```diff
@@ -10,9 +10,9 @@ defmodule Trento.EventHandlersSupervisor do
   @impl true
   def init(_init_arg) do
     children = [
-      Trento.AlertsEventHandler,
-      Trento.RollUpEventHandler,
-      Trento.StreamRollUpEventHandler
+      Trento.Infrastructure.Commanded.EventHandlers.AlertsEventHandler,
```

Comment:
We could alias these to make them shorter

Reply:
done